locations/spiders/childcare_network_us.py

```diff
@@ -12,5 +13,5 @@ class ChildcareNetworkUSSpider(SitemapSpider, StructuredDataSpider):
 
     def post_process_item(self, item, response, ld_data, **kwargs):
         item["name"] = response.xpath("//h2/text()").get()
-
+        apply_category({"amenity": "kindergarten"}, item)
```

Comment:
Ideally `categories.py` should be updated to add `Categories.KINDERGARTEN: {"amenity": "kindergarten"}` to match how categories are applied elsewhere throughout the project.

Reply:
Added to Categories